cloudwatch.tf

```
@@ -0,0 +1,7 @@
+resource "aws_cloudwatch_log_group" "main" {
+  name              = element(var.template_container.*.logConfiguration.options.awslogs-group, 0)
```

```suggestion
  name              = var.template_container[0].logConfiguration.options.awslogs-group
```

Se for sempre zero pode acessar direto. Mas também, se for sempre zero, precisa definir a variável como uma lista?